test/Microsoft.Sbom.Api.Tests/FormatValidator/FormatValidatorTests.cs

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Sbom.Api.FormatValidator;
+using Microsoft.Sbom.Utils;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Sbom.Api.Tests.FormatValidator;
+
+[TestClass]
+public class FormatValidatorTests
+{
+    [TestMethod]
+    public async Task FormatValidator_CanReadValidSbom()
+    {
+        using (var sbomStream = CreateStream(FormatValidatorTestStrings.JsonSuitableForRedaction))
+        {
+            var sbom = new ValidatedSBOM(sbomStream);
+            var rawspdx = await sbom.GetRawSPDXDocument();
+            var details = await sbom.GetValidationResults();
+
+            Assert.AreEqual(FormatValidationStatus.Valid, details.Status);
+            Assert.IsTrue(details.Errors.Count == 0);
+            Assert.IsNotNull(rawspdx);
+            Assert.AreEqual("SPDX-2.2", rawspdx.Version);
+            Assert.AreEqual("CC0-1.0", rawspdx.DataLicense);
+            Assert.AreEqual("sbom-tool 1.0.0", rawspdx.Name);
+            Assert.AreEqual("https://microsoft.com/sbom-tool/alisontest/sbom-tool/1.0.0/cuK7iCCPVEuSmgBfeFPc-g", rawspdx.DocumentNamespace);
```

Comment:
nit: can we search and replace `alisontest` in these strings with something more generic?